package.json

```diff
@@ -97,13 +99,14 @@
     "prettier": "^2.2.1",
     "release-it": "^14.0.3",
     "release-it-lerna-changelog": "^3.1.0",
-    "typescript": "~4.0.3"
+    "typescript": "~4.0.3",
+    "webpack": "^5.68.0"
   },
   "resolutions": {
     "favicons": "5.3.0"
   },
   "engines": {
-    "node": ">= 10.*"
+    "node": "12.* || 14.* || >= 16"
```

Comment:
We can't make this change in the v6 branch. Changing node engines is a breaking change, and as such it's not possible to do.